ServiceNow Service Catalog for Terraform - Edit Workspace updates. (#432)
hashicorp/terraform-docs-common: website/docs/cloud-docs/integrations/service-now/service-catalog-terraform/developer-reference.mdx

| Class Name            | Description                                                |
| --------------------- | ---------------------------------------------------------- |
| `tf_config`           | Helper to pull values from the SN Terraform Configs Table  |

> | `tf_get_workspace`    | Client-callable script to retrieve workspace data          |

Author: I think it's important to leave "client-callable" here because it's a specific type of script in ServiceNow, which makes it different from other backend scripts.

| `tf_http`             | ServiceNow HTTP REST Wrapper for requests to Terraform API |
| `tf_run`              | Resources for Terraform Run API Requests                   |
| `tf_terraform_record` | Manage ServiceNow Terraform Table Records                  |

> | `tf_test_config`      | Client-callable script to test Teraform connectivity       |

Author: Same as above; let's leave "client-callable" as it's a special kind of Script in ServiceNow.

| `tf_util`             | Miscellaneous helper functions                             |
| `tf_variable`         | Resources for Terraform Variable API Requests              |
| `tf_vcs_record`       | Manage ServiceNow Terraform VCS Repositories Table Records |
| `tf_workspace`        | Resources for Terraform workspace API Requests             |

### Example Service Catalog Flows and Actions

-> The ServiceNow Service Catalog for Terraform provides sample Catalog Items that utilize both **Flows** 

> and **Workflows** as their primary process engines. To determine a specific engine used by an Item, open it 

Reviewer: active voice suggestion:
```suggestion
and **Workflows** as their primary process engines. To see which engine an item is using, open it 
```

> in the edit mode and navigate to the **Proces Engine** tab. For instance, **Create Workspace** uses a **Workflow**, 

Reviewer: ```suggestion
in the edit mode and navigate to the **Process Engine** tab. For example, **Create Workspace** uses a **Workflow**, 
```

> whereas **Create Workspace Flow** is built upon a **Flow**. Both can be accessed in the **Studio**. You can also 

Reviewer: ```suggestion
whereas **Create Workspace Flow** is built upon a **Flow**. You can access both in the **Studio**. You can also 
```

> manage **Flows** in the **Flow Designer**. For **Workflows**, navigate to **All > Workflow Editor**. **Flows** are 

Reviewer: ```suggestion
manage **Flows** in the **Flow Designer**. To manage **Workflows**, navigate to **All > Workflow Editor**. **Flows** are 
```

> a newer solution developed by ServiceNow and are generally preferred over **Workflows**.

Reviewer: The final sentence about flows being the perferred solution feels like good content for the beginning of this paragraph. Maybe we should add it to the top of this section?

> The ServiceNow Example Flows can be found in the **ServiceNow Studio > Flows** (or **All > Flow Designer**). 

Reviewer: ```suggestion
You can find the ServiceNow Example Flows in the **ServiceNow Studio > Flows** (or **All > Flow Designer**). 
```

> By default, the Flows execute upon submitting an order request for the various Catalog Items. Admins can modify the 

Reviewer: suggestion for brevity and clarity:
```suggestion
By default, Flows execute when someone submits an order request for a catalog item. Admins can modify the 
```

Reviewer: Looks like the last sentence is left hanging? Should there be more there?

> Flows and Actions to wait on an approval action, include approval rules, and specify approver groups.

Reviewer: Who performs the approval action here? and do the Flows and action have the approval rules? I'm a little confused what this last sentence is conveying.

Author: Admins can define a group of approvers within the organization. I'll change the sentence to add more clarity. Thanks!
